webapp/src/ts/services/browser-detector.service.ts

```diff
@@ -39,6 +39,10 @@ export class BrowserDetectorService {
     });
   }
 
+  public isUsingOutdatedChromeBrowser() {
+    return this.parser.satisfies({ chrome: ">74" }) && !this.parser.satisfies({ chrome: "<90" });
```

Comment:
How are we handling when users have a Chrome 68, for example? What did we decide in that scenario?

Reply:
Also... is it supposed to be 74 or 72? Everywhere in the ticket it mentions 72 (but there may have been other conversations confirming it should be 74).

Reply:
@michaelkohn In this other [issue](https://github.com/medic/cht-core/issues/8456#issuecomment-1677618103) Mr Jones discovered the forever spinner in webview 74. That's why I went with 74 since no version > than 74 has shown the forever spinner.

@latin-panda I think we may need to do extensive testing for Chrome 66-74 to confirm whether we get the forever spinner. What we know for now is we have not seen the forever spinnner for Chrome 75 -90, so for them we just warn them. 

Based on the extensive testing we can come up with the definitive version to completely block and the version to start warning.

Reply:
Cool, thanks @Benmuiruri let's be sure to clarify what our final solution was in the [main issue](https://github.com/medic/cht-core/issues/7770) by editing the first comment.